.github/workflows/test-unit.yml

```diff
@@ -27,6 +27,8 @@ jobs:
       - run: git fetch origin ${{ github.ref }} --depth=100
       - run: git diff origin/master...HEAD --name-only
       - uses: actions/setup-node@v1
+        with:
```

Comment:
Do we need to add this on other workflow files?

Reply:
Nope, the others already have it. This was the only workflow file missing `with`.